### DISCUSSION TITLE
Trying to branch out from main to avoid forking

### DISCUSSION THREAD

**Discussion by Denis-cheboy** [hellocontento/full-stack-assignment-202410#6] [General]
Hello, I have tried to branch out from the main branch to push my code but seems am not permittable to perform that. 
I didnt want to create a fork but seems i have to or is their changes that you can grant me write permissions to just push my code to a feature branch.
Denis-cheboy is my username.

**Comment by onecuriousmindset** [comment]
Hey Denis, 

You actually have to clone the project, keep it in a separate private repo, and once you are done just give Koen ( koen@willow.co ) access to it.

**Reply by Denis-cheboy** [reply]
Thanks for the reply, got it now
